geomstats/geometry/invariant_metric.py

```diff
@@ -1077,3 +1077,33 @@ def inner_product(self, tangent_vec_a, tangent_vec_b, base_point=None):
         inner_prod = self.inner_product_at_identity(
             tangent_vec_a_at_id, tangent_vec_b_at_id)
         return inner_prod
+
+    def parallel_transport(self, tangent_vec_a, tangent_vec_b, base_point):
+        r"""Compute the parallel transport of a tangent vector.
+
+        Closed-form solution for the parallel transport of a tangent vector a
```

Comment:
Do you happen to have a reference for this? I feel we should add more references in the documentation.